### PR TITLE
Add proxy endpoint for the external quality score API

### PR DESCRIPTION
**Pull request by robere2**
* `/api/score/*` will now return score data about a given URL
* URLs are hashed in the database for the sake of privacy
* Database connection code & database table refactored slightly

### CODE REVIEW

File: backend/api.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const express = require("express");
```
Comment (robere2) [inline backend/api.js:1]: More comments needed in this file, probably.

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 node_modules/
 .env
 pgdata/
+/backend/postgres-data/
```
Comment (robere2) [inline .gitignore:3]: `pgdata/` and `/backend/postgres-data/` same purpose?

Comment (robere2) [reply .gitignore:3]: @aidan-lane what is `pgdata/` for?

Comment (aidan-lane) [reply .gitignore:3]: pgdata was a remnant from testing I think, if the volume is called postgres-data then we can remove pgdata

**Comment by aidan-lane** [comment]
LGTM

**Pull request merged**